multispaeti/_multispati_pca.py

```diff
@@ -46,27 +52,24 @@ class MultispatiPCA:
         A distance matrix should be transformed to connectivities by e.g.
         calculating :math:`1-d/d_{max}` beforehand.
 
-    Raises
-    ------
-    ValueError
-        If connectivity is not a square matrix.
-    ZeroDivisionError
-        If one of the observations has no neighbors.
-
     Attributes
     ----------
     components_ : numpy.ndarray
         The estimated components: Array of shape `(n_components, n_features)`.
 
+    eigenvalues_ : numpy.ndarray
+        The eigenvalues corresponding to each of the selected components. Array of shape
```

Comment:
returned components should be the right way of writing this shouldnt it

Reply:
'returned' sounds like return from the function but the components are not returned just stored in an attribute

Reply:
then maybe write kept, selected sounds like selecting components based on a threshold criteiria

Reply:
I am following the conventions used sklearn here again.

e.g. in PCA

> Percentage of variance explained by each of the selected components.